src/SLCore.UnitTests/Listener/LoggerListenerTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using SonarLint.VisualStudio.Core;
+using SonarLint.VisualStudio.SLCore.Core;
+using SonarLint.VisualStudio.SLCore.Listener;
+using SonarLint.VisualStudio.TestInfrastructure;
+
+namespace SonarLint.VisualStudio.SLCore.UnitTests.Listener
+{
+    [TestClass]
+    public partial class LoggerListenerTests
```

Comment:
Why partial class? One is for listener, one is for serialization. 2 different things

Reply:
They are related to logging So I wanted to group them together.

Reply:
But seperate enough so partial class

Reply:
I disagree. We wouldn't have separate classes for SQ client tests then. The 2 things are different. Partial classes are confusing

Reply:
For grouping - use folder/namespace